doc/services/device_mgmt/smp_groups/smp_group_0.rst

```diff
@@ -28,6 +28,8 @@ OS management group defines following commands:
     +-------------------+-----------------------------------------------+
     | ``7``             | OS/Application info                           |
     +-------------------+-----------------------------------------------+
+    | ``8``             | Bootloader parameters                         |
```

Comment:
`Bootloader Information`

Reply:
Seems unaddressed

Reply:
I wonder if group 0 is the right place for this and if group 1 is more suited since it is related to images

Reply:
Fixed